src/features/common/InputTypes/AutoCompleteCountryNew.tsx

```diff
@@ -0,0 +1,379 @@
+/* eslint-disable no-use-before-define */
```

Comment:
Does the name `AutoCompleteCountryNew.tsx` should imply the old component `AutoCompleteCountry.tsx` should be replaced or should both be used next to each other? In the later case I would wonder if`AutoCompleteCountryNew.tsx` maybe could be renamed to better reflect the purpose of its usage?

Reply:
We're updating the dashboard UI page by page. `AutoCompleteCountryNew.tsx` is the component with the new UI and `AutoCompleteCountry.tsx` is the old one - `AutoCompleteCountryNew` will eventually replace `AutoCompleteCountry` in all places, and then will be renamed to `AutoCompleteCountry` (once the old component is no longer useful).